pkg/plugins/policies/meshtrace/api/v1alpha1/validator.go

```diff
@@ -83,8 +83,8 @@ func validateBackend(conf Conf, backendsPath validators.PathBuilder) validators.
 	var verr validators.ValidationError
 	backend := pointer.Deref(conf.Backends)[0]
 	firstBackendPath := backendsPath.Index(0)
-	if (backend.Datadog != nil) == (backend.Zipkin != nil) {
-		verr.AddViolationAt(firstBackendPath, validators.MustHaveOnlyOne("backend", "datadog", "zipkin"))
+	if (backend.Datadog != nil) == (backend.Zipkin != nil) == (backend.OpenTelemetry != nil) {
```

Comment:
I don't think the XOR trick works anymore here, probably needs an explicit `switch` to check if `#(backends != nil)` `== 1`

Reply:
ups :D